packages/release-please/src/release-please.ts

```diff
@@ -527,19 +534,76 @@ const handler = (app: Probot) => {
       url: context.payload.repository.releases_url,
     });
   });
+
   // Check the config schema on PRs.
   app.on(['pull_request.opened', 'pull_request.synchronize'], async context => {
-    const configChecker = new ConfigChecker<ConfigurationOptions>(
-      schema,
-      WELL_KNOWN_CONFIGURATION_FILE
+    const logger = getContextLogger(context);
+    const repoUrl = context.payload.repository.full_name;
+    const baseBranch = context.payload.pull_request.base.ref;
+    const prNumber = context.payload.pull_request.number;
+    const headSha = context.payload.pull_request.head.sha;
+    const defaultBranch = context.payload.pull_request.base.repo.default_branch;
+
+    const headOwner = context.payload.pull_request.head.repo.owner.login;
+    const headRepo = context.payload.pull_request.head.repo.name;
+    const headBranch = context.payload.pull_request.head.ref;
+
+    const schemasByFile: Record<string, object> = {
+      '.github/release-please.yml': schema,
+    };
+    const remoteConfiguration = await getConfig<ConfigurationOptions>(
+      context.octokit,
+      headOwner,
+      headRepo,
+      WELL_KNOWN_CONFIGURATION_FILE,
+      {branch: headBranch}
     );
+
+    // If no configuration is specified,
+    if (!remoteConfiguration) {
+      logger.info(`release-please not configured for (${repoUrl})`);
+      return;
+    }
+    if (!remoteConfiguration.primaryBranch) {
+      remoteConfiguration.primaryBranch = defaultBranch;
+    }
+
+    try {
```

Comment:
Can you add comments for this try-catch block?

Reply:
Done